lib/bolognese/readers/crossref_reader.rb

```diff
@@ -282,18 +282,17 @@ def crossref_people(bibliographic_metadata, contributor_role)
               end
             end.compact
 
-            contributor_type = CR_TO_DC_CONTRIBUTOR_TYPES[a["contributor_role"]]
-
             { "nameType" => "Personal",
               "nameIdentifiers" => name_identifiers,
               "name" => [family_name, given_name].compact.join(", "),
               "givenName" => given_name,
               "familyName" => family_name,
               "affiliation" => affiliation.presence,
-              "contributorType" => contributor_type }.compact
+              "contributorType" => contributor_role == 'author' ? nil : CR_TO_DC_CONTRIBUTOR_TYPES[a["contributor_role"]] }.compact
           else
             { "nameType" => "Organizational",
-              "name" => a["name"] || a["__content__"] }
+              "name" => a["name"] || a["__content__"],
+              "contributorType" => contributor_role == 'author' ? nil : CR_TO_DC_CONTRIBUTOR_TYPES[a["contributor_role"]] }.compact
```

Comment:
Do you anticipate issues with importing Crossref metadata in lupo? Or is it likely that the records would still be created with this logic? I'm not sure if Crossref/"OtherDoi" records are validated in the same way. 

Maybe better to check the tests in lupo once this change is implemented there than trying to anticipate :) But just a thought.

Reply:
@codycooperross - Good point.  It is easy to run the current lupo with the new bolognese gem, so I will run the lupo test suite to make sure there are no problems.  I will also poke around to make sure the right thing happens and that we have the test coverage to expose errors just in case.